### PR TITLE
Mobile chat message info menu

### PR DESCRIPTION
**Pull request by gabriel**
On long press shows, popup menu (alert) with copy text if text message.

Also shows info same as desktop.

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

File: shared/chat/conversation/messages/wrapper.native.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // @flow
 import React, {PureComponent} from 'react'
+import {TouchableHighlight} from 'react-native'
 import {Avatar, Box, Icon, Text} from '../../../common-adapters'
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper.native.js]: same as above, switch to common-adapters/index.native and pull in NativeTouchableHighlight

File: shared/chat/conversations-list/container.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {connect} from 'react-redux'
 import {formatTimeForConversationList} from '../../util/timestamp'
 import {globalColors} from '../../styles'
 import {loadInbox, selectConversation, newChat} from '../../actions/chat'
+import {navigateAppend} from '../../actions/route-tree'
```
Comment (chrisnojima) [inline shared/chat/conversations-list/container.js]: needed?

File: shared/common-adapters/button.native.js [hunk 3]
```diff
@@ -188,4 +189,13 @@ const progress = {
   marginTop: -regularHeight / 2,
 }
 
+export const buttonAlertStyle = {
```
Comment (chrisnojima) [inline shared/common-adapters/button.native.js]: if this is used in one place we should move it out of button

**Comment by keybase-ci-visdiff** [comment]
The commits 77be732043c1...7467f43eeb09 introduce visual changes on linux. <details><summary>:mag_right: 5 changed</summary>

 * Changed: **Popup_Attachment-Popup_Attachment_Message** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup_Attachment-Popup_Attachment_Message-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup_Attachment-Popup_Attachment_Message-7467f43eeb09.png)  
   Popup_Attachment-Popup_Attachment_Message comparison
 * Changed: **Popup_Attachment-Popup_Attachment_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup_Attachment-Popup_Attachment_Downloaded-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup_Attachment-Popup_Attachment_Downloaded-7467f43eeb09.png)  
   Popup_Attachment-Popup_Attachment_Downloaded comparison
 * Changed: **Popup-Following_Valid** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup-Following_Valid-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup-Following_Valid-7467f43eeb09.png)  
   Popup-Following_Valid comparison
 * Changed: **Popup-Following_Revoked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup-Following_Revoked-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Popup-Following_Revoked-7467f43eeb09.png)  
   Popup-Following_Revoked comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Attachment_Popup-Following_Popup_Showing-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488406661061-77be732043c1-7467f43eeb09-linux/Attachment_Popup-Following_Popup_Showing-7467f43eeb09.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 77be732043c1...721f076d6e70 introduce visual changes on linux. <details><summary>:mag_right: 5 changed</summary>

 * Changed: **Popup_Attachment-Popup_Attachment_Message** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup_Attachment-Popup_Attachment_Message-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup_Attachment-Popup_Attachment_Message-721f076d6e70.png)  
   Popup_Attachment-Popup_Attachment_Message comparison
 * Changed: **Popup_Attachment-Popup_Attachment_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup_Attachment-Popup_Attachment_Downloaded-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup_Attachment-Popup_Attachment_Downloaded-721f076d6e70.png)  
   Popup_Attachment-Popup_Attachment_Downloaded comparison
 * Changed: **Popup-Following_Valid** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup-Following_Valid-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup-Following_Valid-721f076d6e70.png)  
   Popup-Following_Valid comparison
 * Changed: **Popup-Following_Revoked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup-Following_Revoked-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Popup-Following_Revoked-721f076d6e70.png)  
   Popup-Following_Revoked comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Attachment_Popup-Following_Popup_Showing-77be732043c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1488407061407-77be732043c1-721f076d6e70-linux/Attachment_Popup-Following_Popup_Showing-721f076d6e70.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

**Comment by gabriel** [comment]
I think ready for review again

**Pull request merged**